src/posix/client.c

```diff
@@ -154,7 +156,7 @@ int main(int argc, char *argv[])
             }
             else
             {
-                write(STDOUT_FILENO, buffer, rval);
+                IgnoreReturnValue(write(STDOUT_FILENO, buffer, rval));
```

Comment:
@bukepo any reason to check the return value?

Reply:
Yes, I think so. It may require multiple writes if the return value is less than `rval`. We can enhance in this PR or later.

Reply:
Besides we also need enable this check in travis.